### PR TITLE
mass TTP changes

### PR DESCRIPTION
**Pull request by bartimus-primed**

### CODE REVIEW

File: ttps/persistence/83a60a83-4d0c-4559-96ca-b5f45438347b.yml [hunk 1]
```diff
@@ -34,3 +34,6 @@ metadata:
   - tr4cefl0w
   tags: []
   enabled: true
+  payloads:
+    package.dll:
```
Comment (privateducky) [inline ttps/persistence/83a60a83-4d0c-4559-96ca-b5f45438347b.yml:38]: missing hashes

**Review by privateducky — CHANGES_REQUESTED** [review]
1 change.
@khyberspache plz check as well.

File: ttps/persistence/83a60a83-4d0c-4559-96ca-b5f45438347b.yml (hunk 1 above)
Comment (bartimus-primed) [reply ttps/persistence/83a60a83-4d0c-4559-96ca-b5f45438347b.yml:38]: They don't exist cause that payload doesn't exist on our server. I believe Alex was working on that

**Review by khyberspache — APPROVED** [review]

File: ttps/persistence/83a60a83-4d0c-4559-96ca-b5f45438347b.yml (hunk 1 above)
Comment (khyberspache) [reply ttps/persistence/83a60a83-4d0c-4559-96ca-b5f45438347b.yml:38]: yeah I'm tracking this. I have his repo forked, but I was waiting for payloads to settle before I add stuff

**Pull request merged**